common/locales/fr.json

```diff
@@ -108,22 +108,37 @@
         "addressStreet": "Rue",
         "addressZipcode": "Code postal",
         "couponKey": "Coupon",
-        "motto": "Devise",
-        "name": "Nom",
-        "title": "Titre"
+        "courseKind": "Type de cours (pour les cours J+S: durée du cours)",
+        "courseNumber": "Numéro de cours",
+        "kind": "Type de camp (chalet, tente, à l'étranger, été, automne)",
```

Comment:
The german and english translation refer with "Unterwegslager" or "traveling" to a camp which is not in a fixed place, but the people sleep every night at a different place.
Deepl says that "à l'étranger" means abroad. (Which would also be a kind of camp, but does not have the same meaning)

Reply:
My bad, a misreading of the deepl translation. I change to use the correct word.